src/containers/SendPage.js

```diff
@@ -508,7 +509,7 @@ class SendPage extends React.Component {
                   <Button
                     modifier='quiet'
                     onClick={
-                      () => this.handleSendValueChange({ target: { value: (this.props.context.value - (this.state.sendFee / 100000000)).toPrecision(8) } })}
+                      () => this.handleSendValueChange({ target: { value: this.props.context.value == 0 ? 0 : ((this.props.context.value * 100000000- 10000)/100000000).toPrecision(8)} })}
```

Comment:
Let's tidy Arizen up one tiny task at a time. I'd really like to move this stray `100000000` into a constant named something like `SATOSHI_MULTIPLIER` so we can simplify things a bit more. It'll never change, but I think we'll sleep a little sounder knowing that the math definitely has the right amount of zero's in every case.